mappings/net/minecraft/world/event/PositionSource.mapping

```diff
@@ -1,5 +1,11 @@
 CLASS net/minecraft/class_5716 net/minecraft/world/event/PositionSource
+	COMMENT A position source is a property of a game event listener.
+	COMMENT
+	COMMENT @see net.minecraft.world.event.listener.GameEventListener#getPositionSource()
 	FIELD field_28184 TYPE_CODEC Lcom/mojang/serialization/Codec;
```

Comment:
```suggestion
	FIELD field_28184 CODEC Lcom/mojang/serialization/Codec;
```
I wanted to do this refactor but forgot, and stupid enigma doesn't allow this refactor.
This is a codec that works for all position sources as long as its type is in the position source registry.